pkg/controller/plan/adapter/ova/builder.go

```diff
@@ -332,15 +332,18 @@ func (r *Builder) mapCPU(vm *model.VM, object *cnv.VirtualMachineSpec) {
 	}
 }
 
-func (r *Builder) mapFirmware(vm *model.VM, object *cnv.VirtualMachineSpec) {
+func (r *Builder) mapFirmware(vm *model.VM, vmRef ref.Ref, object *cnv.VirtualMachineSpec) {
 	var virtV2VFirmware string
 	if vm.Firmware == "" {
 		for _, vmConf := range r.Migration.Status.VMs {
-			if vmConf.ID == vm.ID {
+			if vmConf.ID == vmRef.ID {
 				virtV2VFirmware = vmConf.Firmware
 				break
 			}
 		}
+		if virtV2VFirmware == "" {
+			r.Log.Info("failed to match the vm, model VMid is %s, vmRef ID is %s", vm.ID, vmRef.ID)
```

Comment:
```suggestion
			r.Log.Info("failed to match the vm, model vmID is %s, vmRef ID is %s", vm.ID, vmRef.ID)
```
or `vm id`? this type of `VMid` is unusual

Reply:
yep, should be vmID o vmId

Reply:
yep, should be vmID o vmId
```suggestion
			r.Log.Info("failed to match the vm", "model vmId", vm.ID, "vmRef ID", vmRef.ID)
```